dask/dataframe/tests/test_dataframe.py

```diff
@@ -44,6 +44,24 @@ def test_dataframe_doc():
     assert disclaimer in doc
 
 
+def test_dataframe_doc_from_non_pandas():
+    class Foo:
+        def foo(self):
+            """This is a new docstring that I just made up
+
+            Parameters:
+            ----------
+            None
+            """
+
+    d._bind_operator_method("foo", Foo.foo, original=Foo)
```

Comment:
I hadn't noticed that `d` is not a fixture. This test alters `d` so it makes a different test fail. I tend to think that it is generally preferable to use fixtures when possible. Is there a reason why this test file doesn't use fixtures?

Reply:
I just realized that I am altering the dataframe class not the instance, so that comment is unrelated.